File: find_video.py
from bing_image_downloader import downloader
# Just kidding, I will use Python to scrape a free stock video site like coverr.co or pexels
import urllib.request
import json
req = urllib.request.Request("https://api.pexels.com/videos/search?query=abstract+technology+dark&per_page=3", headers={"Authorization": "563492ad6f9170000100000122e2ef0be66d4826b1f24dffcde10972"})
try:
    with urllib.request.urlopen(req) as response:
        data = json.loads(response.read().decode())
        for video in data['videos']:
            print("Video:", video['id'])
            for file in video['video_files']:
                if file['quality'] == 'hd':
                    print("  URL:", file['link'])
except Exception as e:
    print(e)

File: find_video.js
const https = require('https');
const url = 'https://pixabay.com/api/videos/?key=31245749-bb332ef1cf769a65f97b69c47&q=abstract+dark+technology&per_page=5';

https.get(url, (res) => {
  let data = '';
  res.on('data', chunk => data += chunk);
  res.on('end', () => {
    try {
      const json = JSON.parse(data);
      if (json.hits && json.hits.length > 0) {
        console.log(json.hits[0].videos.large.url);
        console.log(json.hits[1].videos.large.url);
      }
    } catch(e) { console.log("Failed"); }
  });
});
